src/components/Resume/Resume.js

```diff
@@ -254,144 +283,220 @@ const Resume = forwardRef((props, ref) => {
         </div>
       </div>
     ),
-    [sections.skills]: (
+    skills: (
       <div
         key={"skills"}
         className={`${styles.section} ${
-          info.skills?.sectionTitle ? "" : styles.hidden
+          profile?.primary_skills?.length > 0 ||
+          profile?.secondary_skills?.length > 0
+            ? ""
+            : styles.hidden
         }`}
       >
-        <div className={styles.leftSection}>{info.skills?.sectionTitle}</div>
+        <div className={styles.leftSection}>Skills</div>
         <div className={styles.content}>
-          {info?.skills?.points?.length > 0 ? (
-            <ul className={styles.numbered}>
-              {info.skills?.points?.map((elem, index) => (
-                <li className={styles.point} key={`skills-${index}`}>
-                  {elem}
-                </li>
-              ))}
-            </ul>
-          ) : (
-            <span />
-          )}
+          <div className={styles.educationItem}>
+            {profile?.primary_skills?.length > 0 ? (
+              <div>
+                <div className={styles.subtitleHeading}>Primary Skills</div>
+                <ul className={styles.skillNumbered}>
+                  {profile?.primary_skills.map((elem, index) => (
+                    <li className={styles.point} key={"primary" + elem + index}>
+                      {elem}
+                    </li>
+                  ))}
+                </ul>
+              </div>
+            ) : (
+              <span />
+            )}
+            {profile?.secondary_skills?.length > 0 ? (
+              <div>
+                <div className={styles.subtitleHeading}>Secondary Skills</div>
+                <ul className={styles.skillNumbered}>
+                  {profile?.secondary_skills?.map((elem, index) => (
+                    <li
+                      className={styles.point}
+                      key={"secondary" + elem + index}
+                    >
+                      {elem}
+                    </li>
+                  ))}
+                </ul>
+              </div>
+            ) : (
+              <span />
```

Comment:
unnecessary <span /> 
to avoid this, you can use & operator instead of ternary operator.

Reply:
done